examples/LWFA_with_Ionization/include/simulation_defines/param/componentsConfig.param

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2013-2014 Axel Huebl, Heiko Burau, Anton Helm, 
+ *                     Rene Widera, Richard Pausch, Marco Garten
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#pragma once
+
+
+namespace picongpu
+{
+/*! Simulation Starter ---------------------------------------------------
+ *  - defaultPIConGPU         : default PIConGPU configuration
+ */
+namespace simulation_starter = defaultPIConGPU;
+
+/*! Laser Configuration --------------------------------------------------
+ *  - laserNone             : no laser init
+ *  - laserGaussianBeam     : Gaussian beam (focusing)
+ *  - laserPulseFrontTilt   : Gaussian beam with a tilted pulse envelope
+ *                            in 'x' direction
+ *  - laserWavepacket       : wavepacket (Gaussian in time and space, not focusing)
+ *  - laserPlaneWave        : a plane wave
+ */
+namespace laserProfile = laserGaussianBeam;
+
+/*! Field Configuration --------------------------------------------------
+ *  - fieldSolverYee : standard Yee solver
+ *  - fieldSolverLehe: Num. Cherenkov free field solver in a chosen direction
+ *  - fieldSolverDirSplitting: Sentoku's Directional Splitting Method
+ *  - fieldSolverNone: disable the vacuum update of E and B
+ *
+ * * For development purposes: ---------------------------------------------
+ *  - fieldSolverYeeNative : generic version of fieldSolverYee
+ *    (need more shared memory per GPU and is slow)
+ */
+namespace fieldSolver = fieldSolverYee;
```

Comment:
in `cmakeFlags` the field solver is changed for some test cases, but here you set it hard.